src/solvers/radar5.py

```diff
@@ -77,21 +77,21 @@ def __init__(self, problem):
         self.C2M1 = C2-1.0 
         
         # - Statistic values
-        self.statistics["nsteps"]      = 0 #Number of steps
-        self.statistics["nfcn"]        = 0 #Number of function evaluations
-        self.statistics["njac"]        = 0 #Number of Jacobian evaluations
-        self.statistics["njacfcn"]     = 0 #Number of function evaluations when evaluating the jacobian
-        self.statistics["errfail"]     = 0 #Number of step rejections
-        self.statistics["nlu"]         = 0 #Number of LU decompositions
-        self.statistics["nstepstotal"] = 0 #Number of total computed steps (may NOT be equal to nsteps+nerrfail)
+        self.statistics["nsteps"]       = 0 #Number of steps
+        self.statistics["nfcns"]        = 0 #Number of function evaluations
+        self.statistics["njacs"]        = 0 #Number of Jacobian evaluations
+        self.statistics["nfcnjacs"]     = 0 #Number of function evaluations when evaluating the jacobian
+        self.statistics["nerrfails"]    = 0 #Number of step rejections
+        self.statistics["nlus"]         = 0 #Number of LU decompositions
+        # self.statistics["nstepstotal"] = 0 #Number of total computed steps (may NOT be equal to nsteps+nerrfail)
```

Comment:
Should we remove this line since it is a comment?

Reply:
I'd keep this one, might be useful, removed the remaining ones you added comments one.

Reply:
If we keep it I propose we add a comment, my reason is that when I see line 86 above it looks like left-over code that was forgotten and it is not clear if it will break or not if we would remove the hashtag in front.